cli/tests/integrations/test_node.py

```diff
@@ -161,6 +161,16 @@ def test_slave_arg_deprecation_notice():
                    returncode=1)
 
 
+def test_node_ssh_leader_command():
```

Comment:
can we change this to something like `test_node_ssh_with_command`? Since `dcos node ssh` is also a command, the test name is confusing